<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMCDP2DDialog</class>
 <widget class="QDialog" name="FMCDP2DDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="1">
      <widget class="QComboBox" name="cbWavelet">
       <item>
        <property name="text">
         <string>Gauss</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ricker</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Trace Length [ms]:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leDXZ">
       <property name="text">
        <string>10.</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="leF0">
       <property name="text">
        <string>40.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sbLayers">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Number of Layers:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Layers:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leOffset1">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>First Offet [m]:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTableWidget" name="twLayers"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>F0 [hz]:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Wavelet Type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Model Cell Size [m]:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Delay [ms]:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Last Offset [m]:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="leDelay">
       <property name="text">
        <string>0.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Number of Traces:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="sbTraces">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="leOffset2">
       <property name="text">
        <string>2500</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="leTMax">
       <property name="text">
        <string>5000.</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="cbSplitSpread">
       <property name="text">
        <string>Split Spread</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Output Dataset:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leOutput"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FMCDP2DDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FMCDP2DDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
